fullmoon/Localizable/Localizable.xcstrings

```diff
@@ -0,0 +1,522 @@
+          }
+        }
+      }
+    },
+    "keep this screen open and wait for the installation to complete." : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "이 화면을 열어두고 설치가 완료될 때까지 기다리세요."
+          }
+        }
+      }
+    },
+    "Mainframe" : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "메인프레임"
+          }
+        }
+      }
+    },
+    "message" : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "메시지"
+          }
+        }
+      }
+    },
+    "MLX Swift" : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "MLX 스위프트"
+          }
+        }
+      }
+    },
+    "models" : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "모델들"
+          }
+        }
+      }
+    },
+    "new chat" : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "새 채팅"
+          }
+        }
+      }
+    },
+    "new chat with ${prompt}" : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "새 채팅: ${prompt}"
+          }
+        }
+      }
+    },
+    "no chats yet" : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "아직 채팅이 없습니다"
+          }
+        }
+      }
+    },
+    "no results" : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "결과 없음"
+          }
+        }
+      }
+    },
+    "open source" : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "오픈 소스"
+          }
+        }
+      }
+    },
+    "optimized for apple silicon" : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "애플 실리콘에 최적화됨"
+          }
+        }
+      }
+    },
+    "other" : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "기타"
+          }
+        }
+      }
+    },
+    "private" : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "개인 정보 보호"
+          }
+        }
+      }
+    },
+    "runs locally on your device" : {
+      "localizations" : {
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "디바이스에서 로컬로 실행됨"
```

Comment:
```suggestion
            "value" : "기기에서 로컬로 실행됨"
```

큰 문제는 아니지만 위에서 "디바이스" 대신 "기기"를 사용했기 때문에 일관성을 위해 제안드려봅니다.

> It’s not a big issue, but since “기기” was used instead of “디바이스” earlier, I suggest this for consistency.